sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Microsoft.Azure.WebJobs.Extensions.ServiceBus.csproj

```diff
@@ -17,6 +18,13 @@
     <PackageReference Include="Microsoft.Azure.WebJobs" />
     <PackageReference Include="Microsoft.Extensions.Azure" />
     <PackageReference Include="Azure.Messaging.ServiceBus" />
+    <PackageReference Condition="'$(TargetFramework)' == 'net6.0'" Include="Microsoft.Azure.WebJobs.Extensions.Rpc" />
+    <PackageReference Include="Grpc.Tools" VersionOverride="2.51.0" PrivateAssets="all" />
```

Comment:
Needed to roll back to 2.51.0 from 2.58.0 due to this issue - https://github.com/grpc/grpc/issues/32558